src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -60,7 +74,10 @@ public Double averageSumOdd(List<Integer> numbers) {
      * Задача: Выбрать мужчин-военнообязанных (от `fromAge` до `toAge` лет)</p>
      **/
     public List<People> manSelectByAge(List<People> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(people -> people.getAge() > fromAge && people.getAge() < toAge
+                        && people.getSex().equals(People.Sex.MAN))
```

Comment:
It`s possible to use also 
`people.getSex() == People.Sex.MAN`
or 
`people.getSex().equals("MAN")`
but think it`s OK

Reply:
I would first check sex and only then age

Reply:
Done